core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -320,7 +320,7 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
 
         // Make sure the proposed expiration of this request is not post-expiry.
         uint256 requestPassTime = getCurrentTime().add(withdrawalLiveness);
-        require(requestPassTime <= expirationTimestamp, "Request expires post-expiry");
+        require(requestPassTime < expirationTimestamp, "Request expires post-expiry");
```

Comment:
You should make the same change in `requestTransfer`

Reply:
Done.